apps/test/unit/templates/teacherDashboard/EditSectionFormTest.js

```diff
@@ -406,4 +408,78 @@ describe('EditSectionForm', () => {
     const lessonExtrasField = wrapper.find('LessonExtrasField');
     assert.equal(lessonExtrasField.length, 1);
   });
+
+  it('sends completed event when save is clicked', () => {
+    const wrapper = mount(
```

Comment:
Do you need `mount` here to ensure the button calls the appropriate function?

Reply:
I don't actually know what `mount` does, I assumed from its usage elsewhere that's just how you instantiate React components. is there a different way that'd be better here?

Reply:
You can also use `shallow`, which renders just the top level component, whereas `mount` renders all the child component as well. We generally prefer using `shallow` over `mount` but there are cases where `mount` is necessary. I'm not 100% sure but I think this might be one of those cases

Reply:
The tests still pass with `shallow`, so that seems good!